browser/brave_profile_prefs.cc

```diff
@@ -210,6 +210,8 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->SetDefaultPrefValue(
       password_manager::prefs::kPasswordLeakDetectionEnabled,
       base::Value(false));
+  // Restore "Other Bookmarks" migration
+  registry->RegisterBooleanPref(kOtherBookmarksMigrated, false);
```

Comment:
nit - should this go in RegisterProfilePrefsForMigration ?

Reply:
fixed in 07e019e5ed6b0dc01072bcc11f5b9bfa60ef9150